process-migration-service/frontend/package.json

```diff
@@ -13,7 +13,11 @@
     "start": "node src/bin/server.js",
     "test": "jest",
     "precommit": "lint-staged",
-    "locktt": "locktt"
+    "locktt": "locktt",
+    "cypress:run": "cypress run --project ./frontend",
+    "cypress:run-with-dev-server": "start-server-and-test cypress:run'",
```

Comment:
please call this script `test:it`
I assume that start-server-and-test:
* `start` script is invoked by default
* the pim frontend is running on same port as is default for the tool

Can you verify?

Reply:
And we need to invoke this script `test:it` from pom.xml, similarly to how `test` script is invoked. Only difference will be binding it to `integration-test` phase.

Additional task will be checking if failsafe plugin is configured for the project.

Reply:
ok

Reply:
done

Reply:
> please call this script `test:it`
> I assume that start-server-and-test:
> 
> * `start` script is invoked by default
> * the pim frontend is running on same port as is default for the tool
> 
> Can you verify?

Yes updated scripts section and kept only test:it